modules/catalog/src/main/java/org/apache/ignite/internal/catalog/storage/UpdateLogImpl.java

```diff
@@ -189,14 +193,7 @@ static ByteArray updatePrefix() {
         }
     }
 
-    private static class UpdateListener implements WatchListener {
```

Comment:
Why did you make this class non-static?

Reply:
To allow it see the fields of the enclosing class (and avoid creation of same fields in it)

Reply:
It only uses a single fields, I think the previous code was perfectly fine